PKHeX.Core/Resources/text/es/MessageStrings_es.txt

```diff
@@ -148,7 +148,7 @@ MsgSaveChecksumValid = Los checksum son válidos.
 MsgSaveChecksumFailExport = ¿Exportar información del checksum al portapapeles?
 MsgIndexItemRange = Índice de objetos fuera de rango:
 MsgIndexItemGame = No se puede obtener el objeto en el juego:
-MsgIndexItemHeld = Game can't hold item:
+MsgIndexItemHeld = El juego no puede contener el elemento:
```

Comment:
This doesn't make sense at all, when I first localized the file I was skeptical about this line so I left it as it was since I couldn't get up with a proper translation. 

But I guess it could be "El objeto no puede ser equipado en el juego:"